src/es/administration/from_scratch.md

```diff
@@ -2,37 +2,36 @@
+### Importante
 
-### Important
+> Las referencias de los paquetes de software a continuación **son todas basadas en Gentoo** - Hay información en los archivos de docker sobre lo que se requiere en los sistemas tipo debian, y para cualquier otra cosa probablemente podrás ajustar fácilmente como sea necesario.
 
-> Software package references below are all Gentoo-based - there's information in the docker files about what's required on debian-like systems, and for anything else you'll probably be able to easily adjust as necessary.
+Ten en cuenta que la construcción de Lemmy requiere de una gran cantidad de recursos de hardware. Si quieres ejecutar Lemmy en un pequeño VPS con una memoria RAM muy limitada (que parece una forma perfectamente aceptable para ejecutar una instancia de producción), es mejor seguir con la imagen docker, o usar un sistema que tenga más RAM. El uso de RAM es enorme con las builds de Rust.
 
-Note that building Lemmy requires a lot of hardware resources.  If you want to run Lemmy on a small VPS with very limited RAM (which seems like a perfectly acceptable way to run a production instance), maybe stick with the Docker image, or do your code building on another system that has more RAM.  RAM is huge with rust builds.
+Las versiones de etiqueta/lanzamiento incluidas en esta nota provienen de lemmy/docker/prod/docker-compose.yml y estaban actualizadas en el momento en que se creó este documento.  **Definitivamente ajustar a las versiones apropiadas como sea necesario**.
 
-Tag/release versions included in this note come from lemmy/docker/prod/docker-compose.yml and were current at the time this document was created.  **Definitely adjust to appropriate versions as necessary**.
+Tuve que cambiar de usar `sudo` a `su` en algunos lugares ya que el usuario estaba haciendo algo raro/incompleto con el env de sudo para pictrs
 
-I had to switch from using `sudo` to `su`'ing in some places as the user was doing something odd/incomplete with the sudo env for pictrs
+## Configuración
 
-## Setup
-
-| Dependencies                 |                                           |
+| Dependencias                 |                                           |
 |------------------------------|-------------------------------------------|
 | app-admin                    | sudo                                      |
 | dev-vcs                      | [git](https://git-scm.com/)               |
 | dev-lang                     | [rust](https://www.rust-lang.org/)        |
 | dev-db                       | [postgresql](https://www.postgresql.org/) |
 | www-servers                  | [nginx](https://nginx.org/en/)            |
 | sys-apps                     | [yarn](https://yarnpkg.com/)              |
-| (optional) app-accessibility | espeak                                    |
+| (opcional) app-accessibility | espeak                                    |
```

Comment:
I must have forgot to mention this before, we dont use espeak anymore. So you can remove this completely. Otherwise looks good!

Reply:
Done.